### PR TITLE
Modify page scrolling#319

### PR DESCRIPTION
**Pull request by FryDay969**

### CODE REVIEW

**Review by Radomyr-kh — CHANGES_REQUESTED** [review]

**Review by Radomyr-kh — APPROVED** [review]

**Review by MickhaelSh — CHANGES_REQUESTED** [review]
remove package-lock

**Review by MickhaelSh — APPROVED** [review]

File: src/containers/app-content/AppContent.jsx [hunk 1]
```diff
@@ -22,8 +22,8 @@ const AppContent = () => {
       <AppBreadCrumbs />
       <ScrollToTop />
       <AppMain />
-      <ScrollToTopButton />
-      <Footer />
+      {/*<ScrollToTopButton />*/}
+      {/*  <Footer/>*/}
```
Comment (Yurenko) [inline src/containers/app-content/AppContent.jsx:26]: Why did you comment this?

File: src/pages/guest-home-page/GuestHome.jsx [hunk 2]
```diff
@@ -13,33 +13,39 @@ import HowItWorks from '~/containers/guest-home-page/how-it-works/HowItWorks'
...
-        <WhatCanYouDo />
-        <HowItWorks />
-        <WhoWeAre />
-      </Box>
-    </Box>
-  )
+    const {openModal} = useContext(ModalContext)
+    const [searchParams, setSearchParams] = useSearchParams()
+
+    const element = useRef()
```
Comment (Yurenko) [inline src/pages/guest-home-page/GuestHome.jsx:23]: Is it really necessary to use useRef here?

**Comment by Tolik170** [comment]
Fix warnings and run lint

File: src/containers/app-content/AppContent.jsx (hunk 1 above)
Comment (FryDay969) [reply src/containers/app-content/AppContent.jsx]: fixed

File: src/pages/guest-home-page/GuestHome.jsx [hunk 3]
```diff
@@ -31,14 +35,16 @@ const GuestHomePage = () => {
   }, [searchParams, setSearchParams, openModal])
 
   return (
-    <Box data-testid='guestHome' sx={ { flex: 1 } }>
+    <Box data-testid='guestHome' ref={ element } sx={ { flex: 1, overflowY: 'auto' } }>
```
Comment (Radomyr-kh) [inline src/pages/guest-home-page/GuestHome.jsx:38]: You can remove _"data-testid='guestHome'"_ if you don't use it in any test

File: .vscode/settings.json [hunk 4]
```diff
@@ -1,6 +0,0 @@
-{
-  "editor.formatOnSave": false,
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
-  }
```
Comment (Yurenko) [inline .vscode/settings.json:5]: Do not delete this file

File: src/pages/guest-home-page/GuestHome.jsx (hunk 3 above)
Comment (Radomyr-kh) [reply src/pages/guest-home-page/GuestHome.jsx:38]: Also, make sure that all the tests are passing 

image

File: src/pages/guest-home-page/GuestHome.jsx (hunk 2 above)
Comment (FryDay969) [reply src/pages/guest-home-page/GuestHome.jsx]: yes

File: src/pages/guest-home-page/GuestHome.jsx [hunk 5]
```diff
@@ -13,33 +13,39 @@ import HowItWorks from '~/containers/guest-home-page/how-it-works/HowItWorks'
...
-      <Welcome />
-      <Box sx={ { maxWidth: '1128px', margin: '0 auto', overflowX: 'hidden' } }>
-        <FeatureBlock items={ descriptionTimes } />
-        <WhatCanYouDo />
-        <HowItWorks />
-        <WhoWeAre />
-      </Box>
-    </Box>
-  )
+    const {openModal} = useContext(ModalContext)
```
[the remark this answers is not in the document]
Comment (FryDay969) [reply src/pages/guest-home-page/GuestHome.jsx]: Fixed

File: src/pages/guest-home-page/GuestHome.jsx (hunk 3 above)
Comment (FryDay969) [reply src/pages/guest-home-page/GuestHome.jsx:40]: fixed

File: .vscode/settings.json (hunk 4 above)
Comment (FryDay969) [reply .vscode/settings.json]: fixed

**Review by maarekua — APPROVED** [review]

**Pull request merged**